glommio/src/executor.rs

```diff
@@ -559,6 +559,276 @@ impl Default for LocalExecutorBuilder {
     }
 }
 
+/// Specifies a policy by which [`LocalExecutorPoolBuilder`] distributes
+/// [`LocalExecutor`]s on the machine's CPUs / hardware topology.
+#[derive(Debug)]
+pub enum Placement {
+    /// For the `Unbound` variant, the [`LocalExecutor`]s created by a
+    /// [`LocalExecutorPoolBuilder`] are not bound to any CPU.
+    Unbound,
+    /* MaxSpread,
+     * MaxPack,
+     * Custom, */
+}
+
+/// A description of the CPUs location in the machine topology.
+#[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
```

Comment:
I'd be careful with those traits.
What does it mean to say that a location is greater or lesser than another?

If you need to put them on a map that needs a comparison, it is probably better to use one of the `_with` functions to avoid surprising results.

Reply:
Agreed, this is arbitrary.  I used this in parsing `sysfs` but will remove it.